### ISSUE TITLE
Primary key

### ISSUE THREAD

**Issue by mrjoes** [coleifer/wtf-peewee#12]
I have problem with new peewee version and forms generated by wtf-peewee.

In Flask-Admin, there's possibility to work with inline models: when you edit parent, you can also edit related children on the same page. For this, I'm creating model form with allow_pk=True. FormList can contain forms with ID (existing items) and without ID (new items) in one FieldList.

However, due to some recent changes in peewee (or wtf-peewee?), if allow_pk is set to True and ID was not sent back by the client, wtforms validation fails, as ID is marked as required.

Before these recent changes, it was working as expected.

So, to summarise: how wtf-peewee should work with FieldList?
